api/app/controllers/spree/api/v2/storefront/account/addresses_controller.rb

```diff
@@ -67,14 +67,6 @@ def update_service
             def address_params
               params.require(:address).permit(permitted_address_attributes)
             end
-
```

Comment:
@KacperMekarski we can also remove `render_result` from `AddressesController`

Reply:
@damianlegawiec I already removed it from `AddressController` and moved to `BaseController`